app/templates/suppliers/company_summary.html

```diff
@@ -49,27 +49,27 @@
   {% call summary.row(complete=session.get("duns_number", None)) %}
   {{ summary.field_name("DUNS number") }}
   {{ summary.text(session.get("duns_number", "You must answer this question.")) }}
-  {{ summary.edit_link("Edit", url_for(".duns_number")) }}
+  {{ summary.edit_link("Edit", url_for(".duns_number"), hidden_text="DUNS number") }}
   {% endcall %}
   {% call summary.row(complete=session.get("company_name", None)) %}
   {{ summary.field_name("Company name") }}
   {{ summary.text(session.get("company_name", "You must answer this question.")) }}
-  {{ summary.edit_link("Edit", url_for(".company_name")) }}
+  {{ summary.edit_link("Edit", url_for(".company_name"), hidden_text="Company name") }}
   {% endcall %}
   {% call summary.row(complete=session.get("contact_name", None)) %}
   {{ summary.field_name("Contact name") }}
   {{ summary.text(session.get("contact_name", "You must answer this question.")) }}
-  {{ summary.edit_link("Edit", url_for(".company_contact_details")) }}
+  {{ summary.edit_link("Edit", url_for(".company_contact_details"), hidden_text="Contact name") }}
   {% endcall %}
   {% call summary.row(complete=session.get("phone_number", None)) %}
```

Comment:
Yeah I think you're right @karlchillmaid - this should be `email_address`.
Nice spot - that's been in there for over 3 years!